resources/assets/components/blocks/ActionStatsBlock/ActionStatsBlock.js

```diff
@@ -13,22 +14,36 @@ export const ActionStatsBlockFragment = gql`
 `;
 
 const ActionStatsBlock = ({ filterByActionId }) => {
+  const [schoolId, setSchoolId] = useState(null);
   const [schoolLocation, setSchoolLocation] = useState(null);
 
   return (
     <>
       <ActionStatsLeaderboard actionId={filterByActionId} />
 
-      <div className="md:w-1/4 pb-3">
-        <SelectLocationDropdown
-          locationList="domestic"
-          onSelect={event => setSchoolLocation(event.target.value)}
-          selectedOption={schoolLocation || ''}
-        />
+      <div className="flex pb-3">
+        <div className="md:w-1/4 pb-3">
+          <SchoolLocationSelect
+            isClearable
+            onChange={selected =>
+              setSchoolLocation(selected ? selected.value : null)
```

Comment:
This is minor, and potentially all wrong but if we flip this, would we prevent a state refresh / re-render or would it not make a difference since the value doesn't change when it's `null`?
```suggestion
              selected.value ? setSchoolLocation(selected.value) : null
```

Reply:
This is intentional -- we do want to set the `schoolLocation` to `null` in order to get the clearable "X" button working.